src/Module.Server/Common/Models/CrpgAgentApplyDamageModel.cs

```diff
@@ -177,7 +216,93 @@ public override float GetDamageMultiplierForBodyPart(BoneBodyPartType bodyPart,
             case BoneBodyPartType.ShoulderRight:
             case BoneBodyPartType.ArmLeft:
             case BoneBodyPartType.ArmRight:
-                result = !isHuman ? 0.8f : 1f;
+                result = 0.8f;
+                break;
+            case BoneBodyPartType.Legs:
+                result = 0.8f;
+                break;
+        }
+
+        return result;
+    }
+
+    public float CalculateMeleeDamageMultiplierForHumanBodyPart(BoneBodyPartType bodyPart, DamageTypes type)
+    {
+        float result = 1f;
+        switch (bodyPart)
+        {
+            case BoneBodyPartType.None:
+                result = 1f;
+                break;
+            case BoneBodyPartType.Head:
+            case BoneBodyPartType.Neck:
+                switch (type)
+                {
+                    case DamageTypes.Invalid:
+                        result = 2f;
+                        break;
+                    case DamageTypes.Cut:
+                        result = 1.2f;
+                        break;
+                    case DamageTypes.Pierce:
+                        result = 1.6f;
+                        break;
```

Comment:
This was 1.3 before the patch as the calculation was done in another file before

Reply:
I have the second PR ready to make changes to values as requested, this PR was just to improve the format, reduce nesting